### PR TITLE
Fix/controller ad

### LINKED ISSUE #7443: Big Data Cluster fails to load
Azure Data Studio Version: Insiders 10/01
```
Here is the error stack:  TypeError: Cannot read property 'indexOf' of undefined
       at Function.t.getFileName (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:127:42510)
       at t (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:127:41673)
       at Object.<anonymous> (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:175621)
       at a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:313)
       at Object.<anonymous> (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:127:40254)
       at a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:313)
       at Object.<anonymous> (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:127:39831)
       at a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:313)
       at Object.<anonymous> (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:172429)
       at a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:313)
       at Object.i.__awaiter.a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:127:35389)
       at a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:313)
       at Object.i.__awaiter.a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:127:27948)
       at a (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:313)
       at c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:1112
       at Object.<anonymous> (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:1:1122)
       at Object.<anonymous> (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\big-data-cluster\dist\extension.js:213:3)
       at Module.NodeScriptLoader._initNodeRequire.Module._compile (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\out\vs\loader.js:731:45)
       at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
       at Module.load (internal/modules/cjs/loader.js:602:32)
       at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
       at Function.Module._load (internal/modules/cjs/loader.js:533:3)
       at Function.Module._load (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:55009:33)
       at Function.load (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:53118:37)
       at Function.load [as _load] (c:\Users\karlb\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:52971:37)
```

### PR DESCRIPTION
**Pull request by kevcunnane**
- Reapply my previous commit
- add kerberos to excludes in the webpack for the extension

Fixes #7443

### CODE REVIEW

File: extensions/big-data-cluster/extension.webpack.config.js [hunk 1]
```diff
@@ -15,5 +15,6 @@ module.exports = withDefaults({
 		extension: './src/extension.ts'
 	},
 	externals: {
+		kerberos: "commonjs kerberos"
```
Comment (kevcunnane) [inline extensions/big-data-cluster/extension.webpack.config.js]: This is the only change from my previous commit

**Comment by Charles-Gagnon** [comment]
I'd suggest queueing a build and testing it manually before merging this in just to verify

**Comment by kevcunnane** [comment]
Agreed - will follow Charles's suggestion and queue a build.

**Comment by kevcunnane** [comment]
Confirmed working on Mac, will wait for my Windows adhoc build to (hopefully) complete and then test that too.

**Pull request merged**